docs/conf.py

```diff
@@ -75,5 +87,29 @@
 }
 
 
+def autodoc_skip_member(app, what, name, obj, skip, options):
+    # we hide something deliberately
```

Comment:
An annotation like this is not explicit. Better to give a clear explanation.

Reply:
fix